lib/realtime/tenants/connect/backoff.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Realtime.Tenants.Connect.Backoff do
+  @moduledoc """
+  Applies backoff on process initialization.
+  """
+  alias Realtime.RateCounter
+  alias Realtime.GenCounter
+  alias Realtime.Tenants
+  @behaviour Realtime.Tenants.Connect.Piper
+
+  @impl Realtime.Tenants.Connect.Piper
+  def run(acc) do
+    %{tenant_id: tenant_id} = acc
+    connect_backoff_limit = Application.get_env(:realtime, :connect_backoff_limit)
```

Comment:
Should we use `fetch_env!` so there is no way this is returning `nil` because the key is absent?

Reply:
will do